scripts/bootstrap/main.tf

```
@@ -60,3 +60,30 @@ resource "aws_iam_role_policy" "workflow_role_management" {
   role   = module.workflow_role.iam_role_name
   policy = local.rvm_assumption_policy
 }
+
+### IAM Role with OIDC trust for creating break glass role ###
+module "breakglass_role" {
```

This would probably benefit from a conditional for organizations that don't want to have a breakglass-provisioning process this way.

```suggestion
module "breakglass_role" {
  count = var.enable_breakglass_provisioning ? 1 : 0
```

---

If you have this logic in place, none of the breakglass provisioning will work unless you set `enable_breakglass_provisioning` to True